bin/run-db-upload.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import sys
+from os import getenv
+from time import time
+
+import boto3
+from boto3.exceptions import Boto3Error
+
+from db_s3_utils import (
+    get_db_checksum,
+    get_git_sha,
+    get_prev_db_data,
+    set_db_data,
+    JSON_DATA_FILE,
+    DB_FILE,
+)
+
+
+CACHE = {}
+BUCKET_NAME = getenv('AWS_DB_S3_BUCKET', 'bedrock-db-dev')
+
+
+def s3_client():
+    access_key_id = getenv('AWS_DB_ACCESS_KEY_ID')
+    secret_access_key = getenv('AWS_DB_SECRET_ACCESS_KEY')
```

Comment:
How about we set the [environment variables](http://boto3.readthedocs.io/en/latest/guide/configuration.html#environment-variables) that boto uses by default and let it handle reading the credentials from the environment instead?

Reply:
Could do that. Just figuring that we'll likely end up with multiple AWS creds for different things eventually. Can rely on the implicit config until then though I guess.

Reply:
I've updated this with your suggestion.